src/qt/forms/sendcoinsdialog.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>850</width>
-    <height>526</height>
+    <height>572</height>
```

Comment:
I would clean out the unnecessary changes in the .ui file (usually done by Qt Creator).

Reply:
Done